apigo/api/preview.go

```diff
@@ -131,15 +133,29 @@ func Preview(w http.ResponseWriter, r *http.Request, _ httprouter.Params) {
 		return
 	}
 
+	serializedMapping, err := json.Marshal(resourceMapping)
+	if err != nil {
+		http.Error(w, err.Error(), http.StatusBadRequest)
+		return
+	}
+
+	err = storeMapping(serializedMapping, body.ResourceID, body.PreviewID)
```

Comment:
Should we add a removeMapping to avoid storing the mappings of all the previews in redis?

Reply:
good point!

Reply:
done